src/App.tsx

```diff
@@ -13,53 +11,48 @@ import { InlinePlayground } from "./components/InlinePlayground";
 import AppConfig from "./services/configservice";
 import { PLAYGROUND_UI_COLORS } from "./theme";
 import { ThemeProvider } from "@/components/ThemeProvider";
+import {
+  Outlet,
+  RouterProvider,
+  createRouter,
+  createRoute,
+  createRootRoute,
+} from "@tanstack/react-router";
+import { TanStackRouterDevtools } from "@tanstack/react-router-devtools";
 
-export interface AppProps {
-  /**
-   * forTesting indicates whether the app is for testing.
-   */
-  forTesting?: boolean | undefined;
-}
-
-function ForTesting() {
-  return <FullPlayground />;
-}
+const rootRoute = createRootRoute({
+  component: () => (
+    <>
+      <Outlet />
+      <TanStackRouterDevtools />
+    </>
+  ),
+});
```

Comment:
You don't actually need a `component` in a `rootRoute`, but it seemed like a good place to put the dev tools, so I added an outlet and the dev tools here.